language/Turkish/strings.po

```diff
@@ -1700,7 +1700,7 @@ msgstr ""
 
 #: src/frontend/mame/ui/selmenu.cpp:358
 msgid "BIOS selection:"
-msgstr ""
+msgstr "BIOS seçimi"
```

Comment:
Was removing the colon here in the translation intentional?  (It's an honest question, I don't know how punctuation works in Turkish.)

Reply:
added the colon